Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -297,6 +297,8 @@ var ScrollView = React.createClass({
     /**
      * A RefreshControl component, used to provide pull-to-refresh
      * functionality for the ScrollView.
+     *
+     * See [RefreshControl](/react-native/docs/refreshcontrol.html).
```

Comment:
I would use an absolute link so that people opening up the code can click it, but yeah i'm happy with adding links to the docs